dist/css/style.css

```diff
@@ -0,0 +1,198 @@
+html, body {
+  margin: 0;
+  padding: 0;
+  font-family: "Calibri", "Arial", "Helvetica Neue", sans-serif;
+  overflow: hidden;
+}
+
+.page {
+  display: grid;
+  column-gap: 8px;
+  padding: 8px;
+  box-sizing: border-box;
+  height: 100vh;
+  width: 100vw;
+  grid-template-columns: auto 20em;
+  grid-template-rows: 3em auto 2em;
+  grid-template-areas:
+    "header header"
+    "canvas sidebar"
+    "footer footer";
+}
+
+header {
+  grid-area: header;
+  display: flex;
+}
+
+@font-face {
+  font-family: 'Copperplate';
+  src: url('../fonts/Copperplate_V1.woff2')  format('woff');
+}
```

Comment:
Can you remove this and the related `Copperplate_V1.woff2` file since it's no longer used anywhere?